### PR TITLE
Add <div> element

### PR DESCRIPTION
**Pull request by petele**

### CODE REVIEW

**Review by foolip — COMMENTED** [review]
The one we've all been waiting for! @petele can you throw in `compute_from: html.elements.div` so that the status is the earliest version of every browser?

(It's possible that the `HTMLDivElement` interface was originally not there, and we don't need to research that because it doesn't matter.)

**Comment by petele** [comment]
added, ready for review

**Review by captainbrosset — APPROVED** [review]

File: features/div.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
-draft_date: 2024-06-05
 name: <div>
-description: The `<div>` element TODO.
+description: The `<div>` HTML element is the generic container for flow content.
```
Comment (ddbeck) [inline features/div.yml:2]: I'm guessing few developers could give a clear definition of "flow content." Feels a bit like spec speak. What do you think of one of these?

Just being more casual:

```suggestion
description: The `<div>` HTML element is the generic container for content.
```

Or defining the term a bit:

```suggestion
description: The `<div>` HTML element is the generic container for flow content (text and most elements typically found in `<body>`).
```

Comment (petele) [reply features/div.yml]: I went with the more generic since people use div's for more than just text.

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
And it's done! Now people can search for `<div>` and be impressed that we've covered it :D

**Pull request merged**